### PR TITLE
Adds documentation and an example for flink HA

### PR DESCRIPTION
**Pull request by skonto**
This relates to these:
https://github.com/mesosphere/dcos-flink-service/pull/23
https://github.com/mesosphere/universe/pull/1163

### CODE REVIEW

**Comment by joerg84** [comment]
LGTM, testing later today!

File: flink/1.8/README.md [hunk 1]
```diff
@@ -76,6 +79,33 @@ core@ip-10-0-6-55 ~ $ docker run -it mesosphere/dcos-flink:1.2.0-1.2 /bin/bash
...
+
+root@ddfbaadb2094:/flink-1.3.1# wget http://api.hdfs.marathon.l4lb.thisdcos.directory/v1/endpoints/core-site.xml
+
+Add to the conf/flink-conf.yaml file th efollowing options:
+
+high-availability: zookeeper
+high-availability.zookeeper.quorum: master.mesos:2181
+high-availability.zookeeper.storageDir: hdfs://hdfs/flink/recovery/flink
+high-availability.zookeeper.path.root: /dcos-service-flink/flink
+fs.hdfs.hadoopconf: /flink-1.3.1
```
Comment (joerg84) [inline flink/1.8/README.md]: Is this all 1.3.1?

File: flink/1.8/README.md [hunk 2]
```diff
@@ -76,6 +79,33 @@ core@ip-10-0-6-55 ~ $ docker run -it mesosphere/dcos-flink:1.2.0-1.2 /bin/bash
 
 root@2a9c01d3594e:/flink-1.2.0# ./bin/flink run -m <jobmangerhost>:<jobmangerjobmanager.rpc.port> ./examples/batch/WordCount.jar --input file:///etc/resolv.conf --output file:///etc/wordcount_out
 ```
+### Running with HA
```
Comment (EronWright) [inline flink/1.8/README.md]: There are a few spelling mistakes in the below draft.

For clarity would you mind stating that the below is the client configuration.

**Comment by joerg84** [comment]
@skonto could you have a look erons comments?

Otherwise I am happy to merge this after the universe PR.

**Comment by tillrohrmann** [comment]
LGTM modulo Eron's comments.

**Comment by skonto** [comment]
@joerg84 sure I am on vacations so I am might be a bit slow...

**Comment by skonto** [comment]
@joerg84 should I update this PR?

**Comment by skonto** [comment]
closing this as had no response.

**Pull request closed without merging**